zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreter.java

```diff
@@ -123,6 +123,7 @@ public void open() throws InterpreterException {
           try {
             if (!(interpreter instanceof ConfInterpreter)) {
               ((RemoteInterpreter) interpreter).internal_create();
+              ((RemoteInterpreter) interpreter).internal_create();
```

Comment:
Typo?

Reply:
What typo do you mean ?

Reply:
double function call:
`((RemoteInterpreter) interpreter).internal_create();` is called twice

Reply:
Oops，

Reply:
Fixed